debian/changelog

```diff
@@ -1,3 +1,57 @@
+flatpak (0.9.8-1endless1) experimental; urgency=medium
```

Comment:
Can you call this `0endless1`? Then if Debian catches up and we want to import their changes we can actually go forward in version.

Reply:
good point